src/App/Fossa/Ficus/Analyze.hs

```diff
@@ -91,9 +93,10 @@ analyzeWithFicus ::
   ProjectRevision ->
   Maybe LicenseScanPathFilters ->
   Maybe Int ->
+  Maybe FilePath -> -- Debug directory (if enabled)
```

Comment:
[nit] Cases like these are decent for type aliases: `type DebugBundleDir = FilePath`